[1.0] Add Humanoid helper

0b5vr opened the pull request:
### Description

This PR adds `VRMHumanoidHelper` which visualizes humanoid bones.

![image](https://user-images.githubusercontent.com/7824814/172755763-02239249-0e25-445b-8f56-ff298b84ed94.png)

This PR was originally done in #889 but since Humanoid Rig changes are going to be difficult I separated the humanoid helper changes into this PR.

0b5vr committed:
feature (1.0, humanoid): add humanoid helper

nyamadan commented on packages/three-vrm-core/src/humanoid/helpers/VRMHumanoidHelper.ts:
```diff
@@ -0,0 +1,55 @@
+    this._boneAxesMap = new Map();
+
+    Object.values(humanoid.humanBones).forEach((bone) => {
+      if (bone) {
```
ここ、boneはfalsyになることあるんですね。

0b5vr replied:
そんなことないですね。if文外してしまいます。

0b5vr replied:
あ、Type assertionが必要だった。

ただ、新しめのTypeScriptだと必要ないassertionのため、TODOでその旨コメントを残しておきました。

https://github.com/pixiv/three-vrm/pull/987/commits/4fc58ab75172bd2a32126bf87e8264abae3a601d

nyamadan commented on packages/three-vrm-core/src/humanoid/helpers/VRMHumanoidHelper.ts:
```diff
@@ -0,0 +1,55 @@
+    Array.from(this._boneAxesMap.entries()).forEach(([bone, axes]) => {
+      bone.node.updateWorldMatrix(true, false);
+
+      bone.node.matrixWorld.decompose(_v3A, _quatA, _v3B);
```
decomposeしたけど_v3aと_quatAは特に使っていない。

後で直接matrixWorldをコピーしている。

nyamadan reviewed (approved):
レビューしました！この部分の修正だけでしたら再レビューは不要と思います。

https://github.com/pixiv/three-vrm/pull/987/files/908141c13d5c7033268cf55c3fb9384f6dff2fd2#r904470456

0b5vr committed:
refactor (1.0, humanoid): remove an unnecessary if statement

Add a type assertion instead

The type assertion is not needed if we use later versions of TypeScript. Remove it later
